src/features/todo/components/Add/AddTodo.tsx

```diff
@@ -1,9 +1,16 @@
 import TodoComponent from '@/components/Todo/TodoComponent';
+import { addTodoEndpoint } from '../../../../api';
 
 const AddTodo = () => {
-  const handleAddTodo = (todoValue: string) => {
+  const handleAddTodo = async (todoValue: string) => {
     console.log('Adding todo:', todoValue);
-    // Add logic here to actually save the todo
+    try {
+      // Call the addTodoEndpoint function to add todo
+      const result = await addTodoEndpoint(todoValue);
```

Comment:
Just a heads up for this, we use [useMutation](https://tanstack.com/query/latest/docs/framework/react/reference/useMutation) when updating resources on the backend. But no need to this for now since it's a create request. Just change it in the next PR